lib/puppetlabs_spec_helper/module_spec_helper.rb

```diff
@@ -1,3 +1,12 @@
+case ENV['COVERAGE']
+when 'Coveralls'
+  require 'coveralls'
+  Coveralls.wear!
```

Comment:
This needs to setup a exclude for the spec directory, like here https://github.com/puppetlabs/puppetlabs_spec_helper/pull/99/files#diff-c7beddb7111031927788e851096dc9deR38 and here https://github.com/DavidS/puppetlabs-stdlib/commit/d668dd9165699716d5706883b464401f4c0fe146#diff-93830fa29d616f7c87903d08b5b1b29aR14